### PR TITLE
Favorite list

### PR DESCRIPTION
**Pull request by IzumiSy**
🎉

### CODE REVIEW

**Comment by IzumiSy** [comment]
Review ready

**Comment by k0kubun** [comment]
It'd be better to handle favorite event on src/actions/stream.js for this tab.

**Comment by IzumiSy** [comment]
Updated. Looks its working fine with stream?

File: src/components/timeline.js [hunk 1]
```diff
@@ -28,12 +30,14 @@ export default class Timeline extends React.Component {
         <ul className='tabs clearfix'>
           <TweetTabContainer account={this.props.account} tab='home'>Timeline</TweetTabContainer>
           <MentionTabContainer account={this.props.account} tab='mentions'>Mentions</MentionTabContainer>
+          <FavoritesTabContainer account={this.props.account} tab='favorites'>Favorites</FavoritesTabContainer>
```
Comment (IzumiSy) [inline src/components/timeline.js]: >  In my opinion, a frequently updated tab should be placed to left. 

I actually agree to your opinion :+1: The order of tabs in my PR is not really important.
My actual thought on favorite list feature was, like an official app, make it able to see the list from the user profile page. Currently user profile feature is not implemented, so I just put the list as a tab :)

fav

Comment (IzumiSy) [inline src/components/timeline.js]: So to get to the consent, I will push the change on the tab order. Hold on.

Comment (k0kubun) [inline src/components/timeline.js]: I see :+1:

**Comment by k0kubun** [comment]
I found that you can't move tab focus to favorites by shortcut key. Fixed it in https://github.com/k0kubun/Nocturn/commit/ac16bdd26ac7a70c993a70c0cd067ce3b0e7fc58

**Comment by IzumiSy** [comment]
:tada:

**Pull request merged**